src/CMakeLists.txt

```diff
@@ -450,6 +451,26 @@ IF(OpenMVG_USE_ALEMBIC)
 ENDIF(OpenMVG_USE_ALEMBIC)
 
 # ==============================================================================
+# FBX
+# ==============================================================================
+# - optional, builds SfM->FBX point cloud converter
+# ==============================================================================
+IF(OpenMVG_FBX_DIR)
```

Comment:
it would be better to write a proper FindFBX or use one of those on githu bhttps://goo.gl/3jHIga